css/properties/scroll-margin.json

```diff
@@ -15,12 +15,26 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": {
-              "version_added": "68"
-            },
-            "firefox_android": {
-              "version_added": "68"
-            },
+            "firefox": [
+              {
+                "version_added": "90"
+              },
+              {
+                "version_added": "68",
+                  "partial_implementation": true,
```

Comment:
The indentation is off here. The easiest way to fix this is by running `npm run fix`, committing the resulting changes to the file, and pushing to your branch.

Reply:
Thanks! :)